charts/spire/charts/spire-server/templates/statefulset.yaml

```diff
@@ -48,6 +48,7 @@ spec:
       shareProcessNamespace: true
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- if or (gt (len .Values.initContainers) 0) (and .Values.upstreamAuthority.certManager.enabled .Values.upstreamAuthority.certManager.ca.create) }}
```

Comment:
Can the if check on line 53 be removed with this change? Seem duplicate.

Reply:
No still need it, if not running cert-manager with the create. It should only do the extra unit containers.